assets/src/application/services/notification/index.ts

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Icon, notification as defaultNotification } from 'antd';
+import { ArgsProps } from 'antd/lib/notification';
+
+import error from './error';
+
+const getHash = (message): string => {
+	const msg = JSON.stringify(message);
+	return msg
+		.split('')
+		.reduce((a, b) => {
+			a = (a << 5) - a + b.charCodeAt(0);
+			return a & a;
+		}, 0)
+		.toString();
+};
+
+const notification = (type: 'success' | 'info' | 'warning' | 'error' | 'loading') => ({
+	description,
+	duration,
+	message,
+	placement,
+}: ArgsProps): void => {
```

Comment:
might want to have the inner callback return the message key (vitally important if we end up having to use a timestamp) so that consuming code can use it to edit or dismiss notices.

ie:

```
const loadingKey = loadingNotification(loadingNotice);

if (!loading) {
	dismissNotification(loadingKey );
}
```

Reply:
Thx for the suggestion.

Reply:
@tn3rb I have mixed feelings about this `useLoadingToast`, let's pull this outside this PR.

Reply:
The `dismissNotification` is confusing to me because for example if an operation did not finished I shouldn't be able to dismiss that and go ahead with another operation that might be based on the result of first operation.

Reply:
Another option might be `Spin` component, but still not :100: sure about that:
https://ant.design/components/spin/